torch_encoder.py

```diff
@@ -124,19 +83,14 @@ def encode(self, docs: Optional[DocumentArray], parameters: Dict, **kwargs):
         :param kwargs: Additional key value arguments.
         """
         if docs:
-            docs_batch_generator = self._get_docs_batch_generator(docs, parameters)
+            docs_batch_generator = get_docs_batch_generator(
+                docs,
+                traversal_path=parameters.get('traversal_paths', self.default_traversal_path),
+                batch_size=parameters.get('batch_size', self.default_batch_size),
+                needs_attr='text'
```

Comment:
`text`?

Reply:
Fixed in https://github.com/jina-ai/executor-image-torch-encoder/pull/7/commits/e8806b3f2804b23b1e466cabbd6a46f83a613766